config/locales/interface/input_elements/en_supply_biomass.yml

```diff
@@ -94,11 +94,179 @@ en:
         and reprocessing into green gas of natural gas quality.
     energy_greengas_gasification_wet_biomass_share:
       title: Wet biomass gasification (SCW)
-      short_description: 
+      short_description:
       description: Wet gasification is also called Super Critical Water gasification
         (SCW). During SCW wet biomass is converted into green gas under high pressure
         and temperature (at least 375 °C and 221 bar). Above this critical point,
         water achieves supercritical conditions. During this phase all organic molecules
         within the biomass fall apart en reach a new balance in the form of synthesis
         gas (methane, hydrogen, CO<sub>2</sub> and CO) (Gasunie, 2019). Inorganic
         substances crystallize out simultaneously so that they can be recovered.
+    households_diesel_in_crude_oil_share:
+      title: Diesel
+      short_description:
+      description: |
+        Percentage of diesel in oil products used in the domestic sector. This diesel is mainly used
+        for heating and hot water.
+    households_biodiesel_in_crude_oil_share:
+      title: Biodiesel
+      short_description:
+      description: |
+        Percentage of biodiesel in oil products used in the domestic sector. This biodiesel is mainly used
+        for heating and hot water.
+    households_kerosene_in_crude_oil_share:
+      title: Kerosene
+      short_description:
+      description: |
+        Percentage of kerosene in oil products used in the domestic sector. This kerosene is mainly used
+        for heating and hot water.
+    households_bio_kerosene_in_crude_oil_share:
+      title: Bio kerosene
+      short_description:
+      description: |
+        Percentage of bio kerosene in oil products used in the domestic sector. This bio kerosene is
+        mainly used for heating and hot water.
+    households_lpg_in_crude_oil_share:
+      title: Lpg
```

Comment:
Nitpick: in other parts of the model we use all caps ("LPG" instead of "Lpg"). I would be in favour of using all caps here as well.